resources/js/Components/MismatchRow.vue

```diff
@@ -181,7 +187,8 @@ export default Vue.extend({
     .wikit-Link__content {
       word-break: break-word;
     }
-    .wikit-Link.full-description-link {
-      display: inline;
+    .wikit-Button.full-description-button {
+      padding: 0px 2px;
```

Comment:
I am guessing the look of the button has already been discussed.

Reply:
Yes, it has. I'll merge it now, just to get it onto prod - then it can be discussed further. :-)